ddl/column_test.go

```diff
@@ -904,7 +904,7 @@ func (s *testColumnSuite) TestModifyColumn(c *C) {
 		err    error
 	}{
 		{"int", "bigint", nil},
-		{"int", "int unsigned", errUnsupportedModifyColumn.GenByArgs("unsigned true not match origin false")},
+		{"int", "int unsigned", errUnsupportedModifyColumn.GenByArgs("length 10 is less than origin 11")},
```

Comment:
this error message is consistent with MySQL?

Reply:
no, the error is only returned by TiDB because we not support this ddl request yet.